share/hybrid/gpu-manager.c

```diff
@@ -1410,6 +1410,42 @@ static bool create_prime_outputclass(void) {
     return false;
 }
 
+static bool create_offload_serverlayout(void) {
+    _cleanup_fclose_ FILE *file = NULL;
+    _cleanup_free_ char *multiarch = NULL;
+    char command[100];
+    char xorg_d_custom[PATH_MAX];
+
+    snprintf(xorg_d_custom, sizeof(xorg_d_custom), "%s/11-nvidia-offload.conf",
+             xorg_conf_d_path);
+
+    snprintf(command, sizeof(command),
```

Comment:
Why do you need multiarch to create the file?

Reply:
Sorry, I left that in accidentally when I copied the outputclass function. I'll remove it.